src/watertap_contrib/seto/costing/seto_watertap_costing_package.py

```diff
@@ -63,7 +71,12 @@ class SETOWaterTAPCostingData(WaterTAPCostingData):
     def build_global_params(self):
         super().build_global_params()
 
-        self.base_currency = pyo.units.USD_2020
+        if "USD_2021" not in pyo.units._pint_registry:
+            pyo.units.load_definitions_from_strings(
+                ["USD_2021 = 500/708.0 * USD_CE500"]
+            )
+
+        self.base_currency = pyo.units.USD_2021
```

Comment:
I think this is now available in IDAES so maybe you can remove this.  

https://github.com/IDAES/idaes-pse/blob/a1e420575b1f0ffaaf15df2ff5882b8a9e39e260/idaes/core/base/tests/test_costing_base.py#L74so I wonder if this is still needed.

Reply:
We are using IDAES 2.0.0b2, I don't see USD2021 in the units: https://github.com/IDAES/idaes-pse/blob/0bdd637988ede81c1142096b480964759c98467e/idaes/core/base/costing_base.py#L84